server/src/database/seeds/metadata.ts

```diff
@@ -5,8 +5,8 @@ export const seedMetadata = async (prisma: PrismaClient) => {
     'CREATE SCHEMA IF NOT EXISTS workspace_twenty_7icsva0r6s00mpcp6cwg4w4rd',
   );
   await prisma.$queryRawUnsafe(
-    `INSERT INTO metadata.data_source_metadata(
-      id, schema, type, workspace_id
+    `INSERT INTO metadata."dataSourceMetadata"(
```

Comment:
I think it should be dataSource and not dataSourceMetadata

Reply:
It is indeed what we use to instantiate a typeorm DataSource, I'll change that thanks.